avaframe/out3Plot/plotUtils.py

```diff
@@ -490,6 +501,42 @@ def putAvaNameOnPlot(ax, avaDir):
     return infoText
 
 
+def putInfoBox(ax, infoText, location='lowerRight', color='black', hAlignment='right', alphaF=0.5):
+    '''
+    Puts the  in the lower right or upper left or upper right or lowerLeft corner of the given
+    matplotlib axes
+
+    Parameters
+    ----------
```

Comment:
you are missing some parameters and the return
Dont you also need to return the ax?

Reply:
no I don't think so , removed the infoText return